themes/bootstrap5/templates/RecordDriver/DefaultRecord/list-entry.phtml

```diff
@@ -26,6 +26,10 @@
   <?php endif; ?>
   <input type="hidden" value="<?=$this->escapeHtmlAttr($id) ?>" class="hiddenId">
   <input type="hidden" value="<?=$this->escapeHtmlAttr($source) ?>" class="hiddenSource">
+  <?php if (isset($this->overrideId)): ?>
+  <!-- hiddenOverrideId is used for availability checking for non SOLR records -->
```

Comment:
Maybe better to use a PHP comment here so the comment doesn't increase the size of the rendered output sent to the user:

```suggestion
  <?php if (isset($this->overrideId)): // hiddenOverrideId is used for availability checking for non SOLR records?>
```